src/util/text-tracks.js

```diff
@@ -82,11 +82,31 @@ export const addCaptionData = function({
   captionArray.forEach((caption) => {
     const track = caption.stream;
 
-    inbandTextTracks[track].addCue(new Cue(
-      caption.startTime + timestampOffset,
-      caption.endTime + timestampOffset,
-      caption.text
-    ));
+    // in CEA 608 captions, video.js/mux.js sends a content array
+    // with positioning data
+    if (caption.content) {
+      caption.content.forEach((value) => {
+        const cue = new Cue(
+          caption.startTime + timestampOffset,
+          caption.endTime + timestampOffset,
+          value.text
+        );
+
+        cue.line = value.line;
+        cue.align = 'left';
+        cue.position = value.position;
+        cue.positionAlign = 'line-left';
```

Comment:
Forgive my ignorance of captions and tracks, but is `line-left` here going to make us unable to support RTL languages OOTB?

Reply:
This is a good question! My best attempt to answer this will come from [the spec for CEA 608 captions](https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#positioning-in-cea-608), where it states: "CEA-608 does not distinguish between different alignment, so all captions are left aligned and require a WebVTT align cue setting to override the default middle alignment".

My interpretation of this is that there RTL languages _should_ still start at the left by default. If the desire for those tracks was to be right aligned, it is always possible to include that positioning data in the VTT files so it will align right.

Reply:
Let the spec guide us!

Reply:
CEA-608 supports only a limited Latin character set so doesn't have any consideration of RTL languages. 708 however supports Unicode.